### ISSUE TITLE
Make sure all three of us are credited in the about modal, along with mention of the SSE

### ISSUE THREAD

**Issue by nickserv** [wyantb/HashMask#29]

**Comment by wyantb** [comment]
Sure.  But I'm not doing it: anyone who wants to be in the credits should at least not be lazy enough to do it themselves ;)

**Comment by nickserv** [comment]
If you don't mind, I'm reopening this to remind myself to get to this in the next few days.

**Comment by deriachai** [comment]
Credit added in the about pane, if it should be reformatted do so.

Pushed, can be released in 1.1

**Comment by nickserv** [comment]
I tweaked it further, so now all 3 of us are in the list of developers and there are links to the sites for the SSE and RIT. 

Also, I changed my link to my site instead of my email, since I'd rather send people there than get more bot spam to my email address. I have a neat little script I found somewhere that obfuscates email addresses and then turns them into working links once the page is loaded with JavaScript. It's on unreleased versions on my website, so if you'd like me to obfuscate your email links with a similar script, let me know. Otherwise, you guys might be receiving unwanted spam (especially since a copy of this page will be public, and probably indexed by search engines).
